requirements.txt

```diff
@@ -1 +1,3 @@
-Pillow>=10.0.0
\ No newline at end of file
+Pillow
```

Comment:
This change I realise is a bit weird, I didn't want to make the "module_update.py" script too complicated